ESPSTUDIO-11145 [Configuration Management] Enhanced Property Management (#36)
sassoftware/esp-studio-examples: EndToEndExamples/onnx_object_detection/README.md

8. Click **All properties**. 
    - Observe that the **inputrate** field is set to 4, meaning that 4 frames per second are read from the video. 
    - The **publishrate** field determines how many events are sent to the project per second. Because this field does not have a value, it defaults to the value of the **inputrate** field.

>     - The fields **resize_x** and **resize_y** are set to `@PUBLISH_WIDTH@` and `@PUBLISH_HEIGHT@` respectively, which define the resolution of the published frames. The values of these user-defined properties can be found at project level on the left toolbar by clicking <img src="img/properties-icon.png" alt="properties icon" width=3% height=auto/>.

Reviewer: ```suggestion
    - The fields **resize_x** and **resize_y** are set to `@PUBLISH_WIDTH@` and `@PUBLISH_HEIGHT@` respectively, which define the resolution of the published frames. To view the values of these user-defined properties, click <img src="img/properties-icon.png" alt="properties icon" width=3% height=auto/> on the left toolbar.
```
